<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoRefreshDialog</class>
 <widget class="QDialog" name="AutoRefreshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Auto-Refresh Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MIDASDesktopIcons.qrc">
    <normaloff>:/icons/MIDAS_Desktop_Icon.png</normaloff>:/icons/MIDAS_Desktop_Icon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="settingComboBox">
     <item>
      <property name="text">
       <string>Auto-Refresh When Minimized To Tray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Always Auto-Refresh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Never Auto-Refresh</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="refreshLabel1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Refresh Every</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="refreshLabel2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Minutes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AutoRefreshDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AutoRefreshDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
